pkg/importer/dataStream.go

```diff
@@ -853,7 +853,7 @@ func (d *DataStream) isHTTPQcow2() bool {
 }
 
 func (d *DataStream) calculateTargetSize(dest string) int64 {
-	targetQuantity := resource.NewScaledQuantity(util.GetAvailableSpace(filepath.Dir(dest)), 0)
+	targetQuantity := resource.NewScaledQuantity(d.AvailableSpace, 0)
```

Comment:
Maybe this is something I missed in the scratch space PR.  But why is available space being cached?  My concerns are:
1.  One more thing to set when initializing a data stream
2.  If we forget to do above, the value will be zero, so nothing will work
3.  The name is ambiguous.  Free space where?

Reply:
The variable has been there for a while, its the space available in the target pvc. We added this variable when we didn't have scratch space and we calculated the availabe space after we wrote the file there already but before the resize, so we would not resize to the correct size. This is why we set it before doing anything (cached)

Reply:
Maybe rename the variable in your datastream cleanup?

Reply:
I can rename it in this patch, as well, not a big deal not many places use it.